distributed/client.py

```diff
@@ -1831,7 +1848,7 @@ def publish_dataset(self, **kwargs):
         Client.unpublish_dataset
         Client.persist
         """
-        return self.sync(self._publish_dataset, **kwargs)
+        return self.sync(self._publish_dataset, *args, name=name, **kwargs)
```

Comment:
Ugh this isn't valid python 2.  Suggestions?

Reply:
kwargs['name'] = name